File: C programming/Assignment/Assignment5/type4/range.c
#include <stdio.h>

int range(int start,int end);
void main(){
    int start,end;
    printf("Enter the start number:");
    scanf("%d",&start);
    printf("Enter the end number:");
    scanf("%d",&end);
    printf("sum is %d",range(start,end));
}
int range(int start,int end){
    int i,sum=0;
    
    for(i=start;i<=end;i++){
        sum = sum + i;
    }
    return sum;
}

File: Java_programming/basic_java/assignment3/range.java
package assignment3;

public class range {

	public static void main(String[] args) {
		int i,start=1,end=10,sum=0;
//	    printf("Enter the start number:");
//	    scanf("%d",&start);
//	    printf("Enter the end number:");
//	    scanf("%d",&end);
	    
	    for(i=start;i<=end;i++){
	        sum = sum + i;
	    }
	    System.out.printf("%d",sum);

	}

}
